### ISSUE TITLE
`chain_spec_path` should support `https`

### ISSUE THREAD

**Issue by michalkucharczyk** [paritytech/zombienet#615]
For tests with `db_snapshot` stored in cloud, the raw `chain_spec` must be kept together with the snapshot. It would be good to store it in the cloud in the same bucket as `db_snaphot`. 
Would be good if `chain_spec_path` supported the `https` and zombienet automatically downloaded this files (as it downloads db_snapshot file).

**Comment by dzmitry-lahoda** [comment]
is `db_snapshot` also uses https?

**Comment by michalkucharczyk** [comment]
> is `db_snapshot` also uses https?

yes: https://github.com/paritytech/substrate/blob/master/zombienet/0001-basic-warp-sync/test-warp-sync.toml#L14

**Comment by michalkucharczyk** [comment]
and implementation (for k8s): https://github.com/paritytech/zombienet/blob/a1227232ba91b2f6bfd98f54cd820f8b5c0d23dd/javascript/packages/orchestrator/src/providers/k8s/kubeClient.ts#L121
